### PR TITLE
[MacOS] Add ImageMagick

### PR DESCRIPTION
**Pull request by zoran-bjelica**
# Description
Add Imagemagick version 7.1.0

Installation time: 2m 40s

#### Related issue:
https://github.com/actions/virtual-environments/issues/5371

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/macos/software-report/SoftwareReport.Common.psm1 [hunk 1]
```diff
@@ -547,6 +547,11 @@ function Get-YqVersion {
     return "$yqVersion"
 }
 
+function Get-ImageMagickVersion {
+    $imagemagickVersion = Run-Command "magick -version | head -n 1 | cut -d" " -f 2,3"
```
Comment (miketimofeev) [inline images/macos/software-report/SoftwareReport.Common.psm1:551]: we have `Take-Part` function intended for such needs

**Review by ddobranic — APPROVED** [review]

File: images/macos/software-report/SoftwareReport.Common.psm1 (hunk 1 above)
[the remark this answers is not in the document]
Comment (zoran-bjelica) [reply images/macos/software-report/SoftwareReport.Common.psm1]: I have changed it so that it uses Take-Part instead

Comment (zoran-bjelica) [reply images/macos/software-report/SoftwareReport.Common.psm1]: I have changed it so that it uses Take-Part instead

**Review by al-cheb — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

**Pull request merged**